src/general/base_stage/CreatureStageHUDBase.cs

```diff
@@ -676,6 +679,21 @@ public override void PauseButtonPressed(bool buttonState)
     /// </summary>
     public abstract void ShowFossilisationButtons();
 
+    /// <summary>
+    ///   Updates all fossilisation buttons' status of fossilisation
+    /// </summary>
+    public void RefreshFossilisationButtonsFossilStatus()
+    {
+        var fossils = FossilisedSpecies.CreateListOfFossils(false);
+
+        foreach (FossilisationButton button in fossilisationButtonLayer.GetChildren())
+        {
+            var species = button.AttachedEntity.Get<SpeciesMember>().Species;
+            var fossilised = FossilisedSpecies.IsSpeciesAlreadyFossilised(species.FormattedName, fossils);
+            button.AlreadyFossilised = fossilised;
```

Comment:
This doesn't update the tooltip... Compare what happens in `MicrobeHUD.ShowFossilisationButtons`. I think it's probably better to just have a `abstract void UpdateFossilisationButtonStates` in this class and let the specialized HUD classes handle updating the fossilization buttons as they know the entity types etc they should check. And that also allows sharing the common code for setting up the fossilization button between this method and `ShowFossilisationButtons`.